### ISSUE TITLE
BCE Loss or L1 Loss for Depth Loss

### ISSUE THREAD

**Issue by DFLyan** [Megvii-BaseDetection/BEVDepth#200]
As mentioned in your paper, I want to use L1 loss for depth estimation. However, when I only modify F.binary_cross_entropy to F_loss, the algorithm cannot converge. Are there other modifications in the code to reproduce the result in your paper when using L1 loss for depth estimation?
Thanks!
